rastervision/backend/torch_utils/chip_classification/data.py

```diff
@@ -3,19 +3,49 @@
 
 import numpy as np
 import torch
+
 from torch.utils.data import DataLoader, Dataset
+from torch.utils.data.sampler import WeightedRandomSampler
 from albumentations.core.composition import Compose
 from albumentations.augmentations.transforms import (
     Blur, RandomRotate90, HorizontalFlip, VerticalFlip, GaussianBlur,
     GaussNoise, RGBShift, ToGray)
 
 from rastervision.backend.torch_utils.data import DataBunch
-from rastervision.backend.torch_utils.chip_classification.folder import (
-    ImageFolder)
+from rastervision.backend.torch_utils.chip_classification.folder import ImageFolder
 
 log = logging.getLogger(__name__)
 
 
+def calculate_oversampling_weights(imageFolder, rare_classes, desired_prob):
```

Comment:
Camel case shouldn't be used for variable or argument names in Python.

Reply:
You're right, I missed this one.